lightspeed/assemblies/assembly_using-code-bot-for-suggestions.adoc

```diff
@@ -12,14 +12,16 @@ ifdef::context[:parent-context: {context}]
 
 The {AnsibleCodeBot} scans existing content collections, roles, and playbooks hosted in GitHub repositories, and proactively creates pull requests whenever best practices or quality improvement recommendations are available. The bot automatically submits pull requests to the repository, which proactively alerts the repository owner to a recommended change to their content . You can configure {AnsibleCodeBot} to scan your existing Git repositories (both public and private). Your organization must have a valid {LightspeedFullName} subscription to use Ansible code bot. 
 
-{AnsibleCodeBot} uses Ansible lint to recommend code quality improvements. {AnsibleCodeBot} is intended to promote proven practices, patterns, and behaviors while avoiding common errors that can easily lead to bugs or make code harder to maintain. {AnsibleCodeBot} scans your content based on the configured rules, to ensure that your content adheres to Ansible best practices.
+{AnsibleCodeBot} processes your code repositories to recommend code quality improvements. It is intended to promote proven practices, patterns, and behaviors while avoiding common errors that can easily lead to bugs or make code harder to maintain. {AnsibleCodeBot} scans your Ansible content to ensure that it adheres to Ansible best practices.
```

Comment:
The last sentence (beginning with "ansible code bot scans..." repeats what was in the previous two sentences. I would edit to say: 

> Ansible code bot promotes Ansible best practices while avoiding common errors that can lead to bugs or make code harder to maintain. It does this by scanning your code repositories to recommend code quality improvements.